index.md

```diff
@@ -54,8 +54,10 @@ for each level and a very brief introduction to plotting.
 
 ## Data
 
-Data files for the lesson are available here: ({{page.dataurl %}})[{{page.dataurl %}}]
+Data files for the lesson are available here: 
+[{{page.dataurl %}}]({{page.dataurl %}})
 
+Please download the **.csv* (Comma Separated Value) files.
```

Comment:
Ha! I see about the stars :star2: But I think the wildcard needs to be escaped then... :grimacing: Otherwise use `*.csv` instead?